### PR TITLE
Solves issue of stalling audio playback after 5-8 .wav files

### PR DESCRIPTION
**Pull request by AsherThomasBabu**
Solves issue #6

### CODE REVIEW

File: examples/SimpleAudioPlayerZero/SimpleAudioPlayerZero.ino [hunk 1]
```diff
@@ -38,7 +38,6 @@ void setup()
   Serial.println(" done.");
 
   // 44100kHz stereo => 88200 sample rate
```
Comment (per1234) [inline examples/SimpleAudioPlayerZero/SimpleAudioPlayerZero.ino]:
```suggestion
```
You forgot to move this comment along with the ArduinoZero.begin() call.

File: examples/SimpleAudioPlayerZero/SimpleAudioPlayerZero.ino [hunk 2]
```diff
@@ -47,6 +46,7 @@ void loop()
 
   // open wave file from sdcard
   File myFile = SD.open("test.wav");
+  AudioZero.begin(2*44100);
```
Comment (per1234) [inline examples/SimpleAudioPlayerZero/SimpleAudioPlayerZero.ino]:
```suggestion
  // 44100kHz stereo => 88200 sample rate
  AudioZero.begin(2*44100);
```

**Comment by AsherThomasBabu** [comment]
>Please use descriptive commit titles. When someone is browsing the commit history or blame, they should be able to get a good idea of the purpose of the commit just from the title. "Update SimpleAudioPlayerZero.ino" is pretty much useless. We know you updated it, but why? Currently, in order to find out what the commit does, and why, we need to open the commit, read the message, then track down the referenced issue report. Sometimes we might be looking through dozens of commit messages to find the information we need so if we have to do that every time, it makes it an unnecessarily time consuming process.

@per1234 I shall be more descriptive in my future PRs, I am still in the process of learning how to contribute effectively. Thank you for taking the time to patiently tell me how to do things.
>There's a really great article on this subject here:
https://chris.beams.io/posts/git-commit/
very much worth reading.

I will read the article for sure. Also, I have committed the requested changes, so kindly review them.
Thank You
Asher

**Review by aentinger — APPROVED** [review]
LGTM :+1:

**Review by KillzoneKid — COMMENTED** [review]
The stalling is most likely due to memory leak, and how does including non existent method solves the issue?

**Pull request merged**